docs/roles_and_endpoints.md

```diff
@@ -60,8 +60,8 @@ Api for a PISP to implement to allow PISP functionality.
 | Request Challenge                 | Y | `PUT`  | `/consents/{ID}/createCredential`          | ❌ | ✅ | PISP needs to request a challenge for the given consent | 
 | Lookup Consent                    | Y | `GET`  | `/consents/{ID}`                           | ✅ | ✅ |  | 
 | Update Consent                    | Y | `PUT`  | `/consents/{ID}`                           | ✅ | ✅ | To update add a credential or verify a credential, setting the status to `VERIFIED` | 
-| Initiate 3rd party request        | Y | `POST` | `/thirdPartyRequest/transactions`           | ❌ | ✅ |  | 
-| Get 3rd party request information | Y | `GET`  | `/thirdPartyRequest/transactions/{ID}`      | ❌ | ✅ |  | 
+| Initiate 3rd party request        | Y | `POST` | `/thirdPartyRequests/transactions`           | ❌ | ✅ |  | 
```

Comment:
can I check whether the letter "P" should be a capital or small letter?

Reply:
Good catch! Should be small.

Reply:
I think this one not fixed?